includes/discord-application.md

```diff
@@ -29,16 +30,14 @@
     your `HTTP_EXTERNAL` environment variable must be set to `https://tickets.example.com` *(without a trailing `/`)*.
 
 	![Screenshot](/img/discord-application-4.png)
-6. Create the bot user
+6. Navigate to the `Bot` page
 
-	1. Go to the `Bot` page and click `Add Bot`, then `Yes, do it!`
-	2. Click `View Token`, then **copy the token and set it as your `DISCORD_TOKEN` environment variable.**
+	1. Click `View Token`, then **copy the token and set it as your `DISCORD_TOKEN` environment variable.**
+	2. Optionally, disable the "Public Bot" option to prevent other people from adding your bot to their servers. Before you can disable it go to to the `Installation` page and set `Install Link` to `None`. Save changes after which you disable the "Public Bot" in `Bot` page.
```

Comment:
the word 'optionally' should probably be removed, since leaving your bot as public can have pretty major security implications (which admittedly you do touch on) that we don't want to be responsible for

maybe consider something like 'We recommend...' or remove the word entirely